sdcm/utils/common.py

```diff
@@ -142,6 +142,11 @@ def get_job_name():
     return os.environ.get('JOB_NAME', 'local_run')
 
 
+def get_test_name():
+    job_name = get_job_name()
```

Comment:
if there is not job name, try `SCT_CONFIG_FILES` environment variable

Reply:
done